server/container/guice/mailbox-postgres/src/main/java/org/apache/james/modules/mailbox/PostgresMailboxModule.java

```diff
@@ -85,7 +85,6 @@ protected void configure() {
         postgresDataDefinitions.addBinding().toInstance(PostgresMailboxAggregateModule.MODULE);
 
         install(new PostgresQuotaModule());
-        install(new PostgresQuotaSearchModule());
```

Comment:
Could you confirm quota search is well implemented with OpenSearch?

Reply:
It is cause when I had this, test were failing on a double injection?

Reply:
To be a bit more precise they are being injected by the module chooser